css/src/components/modal.css

```diff
@@ -71,17 +95,33 @@
 
 .Modal-footer {
   display: flex;
+  align-items: center;
+  height: 80px;
+  width: 100%;
+  box-sizing: border-box;
+  padding-left: var(--spacing-xl);
+  padding-right: var(--spacing-xl);
```

Comment:
Revert changes to:
```
padding: var(--spacing-xl);
```

Reply:
Height of footer is fixed (80px). Hence it is better to just give right and left padding on a `flex` container.